test/parallel/test-net-socket-local-address.js

```diff
@@ -20,12 +20,6 @@ const server = net.createServer(function(socket) {
 
 const client = new net.Socket();
 
-server.on('close', common.mustCall(function() {
-  assert.deepEqual(clientLocalPorts, serverRemotePorts,
-                   'client and server should agree on the ports used');
-  assert.equal(2, conns);
-}));
```

Comment:
Not as the test was written. Most of the times, it would run once, but every once in a while on Windows (and possibly elsewhere, but definitely on Windows), it would run twice.

In [a stress test run](https://ci.nodejs.org/job/node-stress-single-test/334/nodes=win-vs2015/console), the test ran 9999 times on Windows. 9988 times, the function ran once. But 11 times, the test failed because it ran twice.